### PR TITLE
Wizard recipe: SEAL-v3.5.4

### PR DESCRIPTION
**Pull request by sloede**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: SEAL
* Version: v3.5.4

@staticfloat please review and merge.

### CODE REVIEW

File: S/SEAL_jll/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SEAL_jll"
```
Comment (giordano) [inline S/SEAL_jll/build_tarballs.jl:5]: This should be
```suggestion
name = "SEAL"
```
The `_jll` is automatically appended in the JLL package.  Note that you must also rename the directory to exactly match this name

Comment (sloede) [reply S/SEAL_jll/build_tarballs.jl]: I am very sorry about that, this clearly was an oversight... Fixed in 2a36fd1.

**Comment by sloede** [comment]
The build error seems to be in the following [line](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=4222&view=logs&j=927318f7-5baa-51a9-d154-f0e151de80b6&t=61c53072-7fae-54a4-3627-e9338c261c59&l=1138):
```
ERROR: could not load library "/agent/_work/1/s/S/SEAL/build/x86_64-linux-gnu-cxx11/yNFKJywe/destdir/lib/libsealc.so"
/bin/../lib/julia/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by /agent/_work/1/s/S/SEAL/build/x86_64-linux-gnu-cxx11/yNFKJywe/destdir/lib/libsealc.so)
```

I have seen a similar issue in my local installation as well. It seems like the libstdc++ version of the Yggdrasil build system and the one shipped in Julia do not match. On my machine, I was able to fix it by replacing `/bin/../lib/julia/libstdc++.so.6` with my system version of `libstdc++.so.6`. However, what can I do here?

**Comment by sloede** [comment]
OK, it seems like reducing the preferred GCC version to 7.1.0 (which matches the libstdc++ shipped with Julia) did the trick...

**Pull request merged**